state/backups/backups_test.go

```diff
@@ -102,7 +102,8 @@ func (s *backupsSuite) TestCreateOkay(c *gc.C) {
 
 	// Test the call values.
 	s.Storage.CheckCalled(c, "spam", meta, archiveFile, "Add", "Metadata")
-	filesToBackUp, _ := backups.ExposeCreateArgs(received)
+	backupDir, filesToBackUp, _ := backups.ExposeCreateArgs(received)
+	c.Check(backupDir, gc.Equals, "")
```

Comment:
This isn't really sufficient as it will still pass even if the code has a mistake and doesn't properly use the backup dir. We need to set the backup dir in backups.Paths{} above to a non-empty value

Reply:
I have replaced, the original default empty string, with the creating of a temporal directory using `c.Mkdir()`

Reply:
The place where you made the change is in TestLegacy which is not this test. You need to change how paths is set up on line 95